.github/workflows/ci.yml

```diff
@@ -52,10 +60,10 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Install Python 3.10
+      - name: Install Python 3.13
         uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
+          python-version: 3.13
```

Comment:
it's probably best to keep this as a string in case future versions end with one or more zeroes.

Reply:
additionally it looks like 16.x will need to stick to python 3.10.

Reply:
Thoughts on these changes?